create_user_interface/User.cs

```diff
@@ -0,0 +1,97 @@
+using System;
+
+/**
+ * Class for simulating users
+ * \author: Nate Hughes <njh2986@vt.edu>
+ */
+
+namespace Hats
+{
+
+	namespace User
+	{
+
+		public class User
+		{
+
+			/** A constructor.
+			* @param username is the value that will set the internal username
+			* @param password is the value that will set the internal password
+			**/
+			User(string username, string password){
```

Comment:
It seems like we're missing configuration information for where this simulated user will be sending commands. Also, the username and password don't seem like longer term parameters - only relevant for a login attempts.